cabal.project

```diff
@@ -176,8 +176,7 @@ allow-newer:
   , servant-openapi3:*
 
 constraints:
-    base == 4.18.2.0
```

Comment:
Pinning base was useful when updating the index state, but we can always add back later.